app/Http/Controllers/Admin/Download/DownloadActivityController.php

```diff
@@ -321,4 +327,20 @@ public function sanitizeRequest($request): array
 
         return $queryParams;
     }
+
+    /**
+     * Returns complete Humanitarian scope vocabulary
+     * Example 1-2-Glide.
+     *
+     * @param array $humanitarianScopeVocabularyArray
+     * @param string $humanitarianScopeNumericIndex
+     *
+     * @return string
+     */
+    private function getCompleteHumanitarianScopeVocabulary(array $humanitarianScopeVocabularyArray, string $humanitarianScopeNumericIndex):string
```

Comment:
check for null value. not sure if this code covers that case

Reply:
Works even if Humanitarian scope vocabulary is empty in data. the function return "-" in such case